services/copyright-service/src/services/similarity.service.js

```
@@ -1,305 +1,735 @@
+      switch (ext) {
+        case '.pdf':
+          const pdfData = await pdf(buffer);
+          return pdfData.text || '';
+        case '.docx':
+          const docxResult = await mammoth.extractRawText({ buffer });
+          return docxResult.value || '';
+        case '.doc':
+          // For .doc files, try to extract as text (basic approach)
+          // Note: For better .doc support, you might need a library like 'textract' or 'antiword'
+          try {
+            const text = buffer.toString('utf8', 0, Math.min(buffer.length, 10000));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)